custom-connectors/CopilotForFinanceCommunications/apiDefinition.swagger.json

```diff
@@ -1,765 +1,1376 @@
+            "name": "x-copilotforfinace-api-version",
+            "description": "Specifies the API version.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "language",
+            "description": "The language code is in BCP 47 format.",
+            "type": "string"
+          }
+        ],
         "responses": {
           "200": {
-            "description": "Success.",
+            "description": "The request has succeeded.",
             "schema": {
+              "required": [
+                "data"
+              ],
               "type": "object",
               "properties": {
-                "id": {
-                  "type": "string",
-                  "description": "Chat thread ID.",
-                  "x-ms-summary": "ID"
-                },
-                "topic": {
-                  "type": "string",
-                  "description": "Chat thread topic.",
-                  "x-ms-summary": "Topic"
-                },
-                "createdOn": {
-                  "type": "string",
-                  "description": "Time when the thread was created.",
-                  "x-ms-summary": "Created On"
-                },
-                "deletedOn": {
-                  "type": "string",
-                  "description": "Time when the thread was deleted.",
-                  "x-ms-summary": "Deleted On"
-                },
-                "createdByCommunicationIdentifier": {
-                  "type": "object",
-                  "properties": {
-                    "rawId": {
-                      "type": "string",
-                      "description": "Raw ID of the identifier.",
-                      "x-ms-summary": "Raw ID"
-                    },
-                    "communicationUser": {
-                      "type": "object",
-                      "properties": {
-                        "id": {
-                          "type": "string",
-                          "description": "ID of the communication user.",
-                          "x-ms-summary": "ID"
-                        }
-                      },
-                      "description": "Communication user."
-                    }
-                  },
-                  "description": "Communication identifier of creator."
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/StatusDefinition"
+                  }
                 }
               }
             }
           },
-          "401": {
-            "description": "Unauthorized"
-          },
-          "403": {
-            "description": "Forbidden"
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
           }
-        },
-        "operationId": "GetThreadProperties",
```

Comment:
The operation has been deleted. This change breaks existing flows. ValuePath: operation/GetThreadProperties

Reply:
We are adding a new template for a custom connector, so nothing is actually deleted.
Endpoint is deleted compared to prior iteration because prior iteration had incorrect swagger included.